server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -1191,11 +1145,20 @@ public boolean isPathExist(PartialPath path) {
     return mtree.isPathExist(path);
   }
 
+  protected IMeasurementMNode getMeasurementMNode(PartialPath fullPath) throws MetadataException {
+    return (IMeasurementMNode) getNodeByPath(fullPath);
+  }
+
   /** Get node by path */
   public IMNode getNodeByPath(PartialPath path) throws MetadataException {
     return mtree.getNodeByPath(path);
   }
 
+  public IMeasurementSchema getMeasurementSchemaByPath(PartialPath fullPath)
+      throws MetadataException {
+    return ((MeasurementMNode) IoTDB.metaManager.getNodeByPath(fullPath)).getSchema();
+  }
+
```

Comment:
There's already a equivalent method in MManager, 
public IMeasurementSchema getSeriesSchema(PartialPath fullPath) throws MetadataException
You could use this instead without adding new method.